doc/weakly_supervised.rst

```diff
@@ -149,94 +147,78 @@ Or alternatively (using a preprocessor):
-       [-3.61531114e+01,  4.86778289e-07,  2.12654397e-08]])
-
-.. warning::
-    
-    If you try to use `transform` with a similarity learner, an error will
-    appear, as you cannot transform the data using them.
+>>> score = mmc.pair_score([[[3.5, 3.6], [5.6, 2.4]], [[1.2, 4.2], [2.1, 6.4]], [[3.3, 7.8], [10.9, 0.1]]])
+>>> score
+array([-0.49627072, -3.65287282, -6.06079877])
 
-Also, as explained before, our metric learner has learned a distance between
-points. You can use this distance in two main ways:
+This is useful because ``pair_score`` matches the **score** semantic of 
+scikit-learn's `Classification metrics
+<https://scikit-learn.org/stable/modules/model_evaluation.html#classification-metrics>`_.
 
-- You can either return the distance between pairs of points using the
-  `pair_distance` function:
+For metric learners that learn a distance metric, there is also the ``pair_distance``
+method.
 
 >>> mmc.pair_distance([[[3.5, 3.6, 5.2], [5.6, 2.4, 6.7]],
 ...                  [[1.2, 4.2, 7.7], [2.1, 6.4, 0.9]]])
 array([7.27607365, 0.88853014])
 
-- Or you can return a function that will return the distance (in the new
-  space) between two 1D arrays (the coordinates of the points in the original
-  space), similarly to distance functions in `scipy.spatial.distance`. To
-  do that, use the `get_metric` method.
+.. warning::
+    
+    If you try to use ``pair_distance`` with a bilinear similarity learner, an error
+    will be thrown, as it does not learn a distance.
+
+You can also return a function that will return the metric learned. It can
+compute the metric between two 1D arrays, similarly to distance functions in
+`scipy.spatial.distance`. To do that, use the ``get_metric`` method.
 
 >>> metric_fun = mmc.get_metric()
 >>> metric_fun([3.5, 3.6, 5.2], [5.6, 2.4, 6.7])
 7.276073646278203
 
-- Alternatively, you can use `pair_score` to return the **score** between
-  pairs of points (the larger the score, the more similar the pair).
-  For Mahalanobis learners, it is equal to the opposite of the distance.
+You can also call ``get_metric``` with bilinear similarity learners, and you will get
+a function that will return the similarity between 1D arrays.
 
->>> score = mmc.pair_score([[[3.5, 3.6], [5.6, 2.4]], [[1.2, 4.2], [2.1, 6.4]], [[3.3, 7.8], [10.9, 0.1]]])
->>> score
-array([-0.49627072, -3.65287282, -6.06079877])
-
-  This is useful because `pair_score` matches the **score** semantic of 
-  scikit-learn's `Classification metrics
-  <https://scikit-learn.org/stable/modules/model_evaluation.html#classification-metrics>`_.
+>>> similarity_fun = algorithm.get_metric()
+>>> similarity_fun([3.5, 3.6], [5.6, 2.4])
+-0.04752
 
-For similarity learners `pair_distance` is not available, as they don't learn
-a distance. Intead you use `pair_score` that has the same behaviour but
-for similarity.
+Finally, as explained in :ref:`mahalanobis_distances`, these are equivalent to the Euclidean
```

Comment:
same question here

Reply:
Same, its rendered as "Mahalanobis distances", so its a bit redundant to repeat the noun